src/vtkF3DWin32OutputWindow.cxx

```diff
@@ -62,16 +65,21 @@ void vtkF3DWin32OutputWindow::DisplayText(const char* someText)
 
   HWND hWnd = static_cast<HWND>(this->EditControlHandle);
 
-  int index = GetWindowTextLength(hWnd);
-  SetFocus(hWnd);
-
   // select end of text
+  int index = GetWindowTextLength(hWnd);
   SendMessageW(hWnd, EM_SETSEL, (WPARAM)index, (LPARAM)index);
 
   // print text
   SendMessageW(hWnd, EM_REPLACESEL, 0, reinterpret_cast<LPARAM>(wstr.c_str()));
 
-  // print text to cout for testing purpose
-  std::cout<<someText;
+  // still print text to std::cout in
+  // order to be able to test verbose outputs
+  std::cout<<text;
   std::cout.flush();
 }
+
+//------------------------------------------------------------------------------
+void vtkF3DWin32OutputWindow::WaitForUser()
+{
+  MessageBoxW(nullptr, L"Press Ok to continue.", L"Information", MB_OK);
```

Comment:
Not a big fan of this approach, an alternative is as follows:
https://gitlab.kitware.com/paraview/paraview/-/merge_requests/5383/diffs

Wdyt @jpouderoux @Meakk ?

Reply:
I think `MessageBox` is fine.

Reply:
First parameter should be outputwindows handle.

Reply:
But LOL, the handle is stored in a hidden global variable...

Reply:
So: 
`
HWND hCurWnd = FindWindowExA(nullptr, hCurWnd, "vtkOutputWindow", nullptr);
MessageBoxW(hCurWnd, L"Press OK to continue.", L"F3D", MB_ICONINFORMATION|MB_OK);`

Reply:
Can he just use the handle of the edit control? The problem with FindWindow is it can return window of a different process.

Reply:
Does the owner handle actually matters here ? in VTK, the PrompText method just do not care.

Reply:
The doc is a bit unclear: https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-messagebox?redirectedfrom=MSDN

It seems to matter in the HELP context, not much in other context.

Reply:
@Meakk @jpouderoux : any inputs here ?

Reply:
Sorry idk :)

Reply:
Sure @mwestphal just pass it null then.